### PR TITLE
bump jruby 1.7.21

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

**Comment by yosifkit** [comment]
changeset: https://github.com/cpuguy83/docker-jruby/compare/a108cb09f3213b8230242bbace8880174f81a6f0...10d7ac08b7138c274a819af1bd8258d61cb55f9e

LGTM, Build test of #875; 9ad484abdc63d0b044f79f708a5e67d2e25fab5b (`jruby`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/9ad484abdc63d0b044f79f708a5e67d2e25fab5b/library/jruby"
$ bashbrew build "$url"
Cloning jruby (git://github.com/cpuguy83/docker-jruby) ...
Processing jruby:1.7 ...
Processing jruby:1.7.21 ...
Processing jruby:latest ...
Processing jruby:1.7-jre ...
Processing jruby:1.7.21-jre ...
Processing jruby:1.7-jdk ...
Processing jruby:1.7.21-jdk ...
Processing jruby:1.7-onbuild ...
Processing jruby:1.7.21-onbuild ...
Processing jruby:9.0.0.0 ...
Processing jruby:9.0.0.0-jre ...
Processing jruby:9.0.0.0.rc1 ...
Processing jruby:9.0.0.0.rc1-jre ...
Processing jruby:9.0.0.0-jdk ...
Processing jruby:9.0.0.0.rc1-jdk ...
Processing jruby:9.0.0.0-onbuild ...
Processing jruby:9.0.0.0.rc1-onbuild ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing jruby:1.7
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:1.7-jdk
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:1.7.21-jdk
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:1.7-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:9.0.0.0
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:9.0.0.0-jdk
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
testing jruby:9.0.0.0-onbuild
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'ruby-standard-libs' [4/6]...passed
	'ruby-gems' [5/6]...passed
	'ruby-bundler' [6/6]...passed
```

File: library/jruby [hunk 1]
```diff
@@ -1,21 +1,17 @@
 # maintainer: Brian Goff <cpuguy83@gmail.com> (@cpuguy83)
 
-1.7: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
-1.7.20: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
-1.7.20.1: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
+1.7: git://github.com/cpuguy83/docker-jruby@10d7ac08b7138c274a819af1bd8258d61cb55f9e 1.7/jre
+1.7.21: git://github.com/cpuguy83/docker-jruby@10d7ac08b7138c274a819af1bd8258d61cb55f9e 1.7/jre
 latest: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
```
Comment (tianon) [inline library/jruby]: uhhhh

File: library/jruby [hunk 2]
```diff
@@ -1,21 +1,17 @@
...
 
-1.7-jre: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
-1.7.20-jre: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
-1.7.20.1-jre: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jre
+1.7-jre: git://github.com/cpuguy83/docker-jruby@10d7ac08b7138c274a819af1bd8258d61cb55f9e 1.7/jre
+1.7.21-jre: git://github.com/cpuguy83/docker-jruby@10d7ac08b7138c274a819af1bd8258d61cb55f9e 1.7/jre
 
 1.7-jdk: git://github.com/cpuguy83/docker-jruby@a108cb09f3213b8230242bbace8880174f81a6f0 1.7/jdk
```
Comment (tianon) [inline library/jruby]: uhhhh

**Comment by cpuguy83** [comment]
Who needs those tags anyway.... :)

Updated.

**Comment by tianon** [comment]
LGTM :+1:

**Pull request merged**